src/super_context.ml

```diff
@@ -79,6 +80,9 @@ let installed_libs t = t.installed_libs
 
 let find_scope_by_dir t dir = Scope.DB.find_by_dir t.scopes dir
 let find_scope_by_name t name = Scope.DB.find_by_name t.scopes name
```

Comment:
This functions and `Scope.DB.find_by_name` are a bit dodgy now. I understand that we are keeping them around only for the old ppx stuff, right? If so, lets add a `DUNE2` comment to remember to delete them.

Reply:
BTW, the `Scope.DB.find_by_name` call in `create` is no longer correct.

Reply:
In fact the `List.hd` in `Scope.DB.find_by_name` seems too dodgy. We should make this function return a list and make the call site fail if the list contains more than one element.